### ISSUE TITLE
[BUG] cant install mubeng from github

### ISSUE THREAD

**Issue by vay3t** [kitabisa/mubeng#88]
**Describe the bug**

I cant install mubeng from github

**To Reproduce**

```bash
go install github.com/kitabisa/mubeng/cmd/mubeng@latest
```

![image](https://user-images.githubusercontent.com/6080369/146661190-89684caa-e131-4cc2-a0ea-1e8ac34228ad.png)

**Expected behavior**

Install mubeng from github

**Environment (please complete the following information):**

- OS: Ubuntu 20.04
- OS version: Linux nanopineo3 5.10.63-rockchip64 aarch64 GNU/Linux

**Comment by dwisiswant0** [comment]
Yep, currently the package URL is down. You should try building from source.

**Comment by vay3t** [comment]
it is necessary to change line 1 https://github.com/kitabisa/mubeng/blob/master/go.mod#L1 by github.com/kitabisa/mubeng
